src/snowflake/cli/api/project/schemas/entities/streamlit_entity_model.py

```diff
@@ -17,14 +17,14 @@
 from typing import List, Literal, Optional
```

Comment:
Should we put models in one module?

Reply:
I believe we will eventually move each entity to its place under the existing structure, but for now it's easier to have these grouped together.